template/src/tsconfig.json

```diff
@@ -1,8 +1,6 @@
 {
   "extends": "../tsconfig.json",
   "compilerOptions": {
-    "module": "es2015",
-    "rootDir": "./",
-    "outDir": "../dist"
+    "rootDir": "./"
```

Comment:
so we are doing ```CommonJS```?

Reply:
Imo. we should use the same module type for production, staging, local and unit tests.
Due to my previous experiences esm modules can cause problems when modules/files have initialization logic in 1-st level (without any wrapper function).

Not resovling comment, please write any pros/cons below if something will came into your mind.